src/main/resources/lang/english.lang

```diff
@@ -238,6 +238,11 @@ enchantments:
 		# It's sweeping on 1.10 and below for some reason.
 		sweeping: Sweeping Edge
 		sweeping_edge: Sweeping Edge
+		# New 1.13 Enchantments
+		channeling: Channeling

```

Comment:
channeling: Channeling, Channelling

Supporting British English ❤️ (My native language isn't even English)

Reply:
Ill quickly whip that change up... I just went with the actual name of the enchantment, but I see what you mean there!

Reply:
Changes Complete